pyproject.toml

```
@@ -31,8 +31,8 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-cdflib = "==1.3.1"
-imap-data-access = ">=0.5.0"
+cdflib = ">=1.3.1"
```

Do we want to make this a karat dependency instead? That way, we won't get unexpected major version updates, since they tend to need some code changes.

```suggestion
cdflib = "^1.3.1"
```